downloader/src/main/kotlin/vcs/GitRepo.kt

```diff
@@ -189,17 +189,15 @@ class GitRepo : VersionControlSystem(), CommandLineTool {
             runRepo(workingTree.workingDir, *syncArgs.toTypedArray())
 
             log.debug { runRepo(workingTree.workingDir, "info").stdout }
-
-            true
-        } catch (e: IOException) {
-            e.showStackTrace()
+        }.onFailure {
+            it.showStackTrace()
```

Comment:
Consider renaming `it` to `e`, same above.

Reply:
I actually thought about this when @oheger-bosch did a [similar change](https://github.com/oss-review-toolkit/ort/pull/3894/files#diff-439602bba9d79021802c46707b56c056fdb9f49373dac17ba938cdc4552605cdR88), but in exactly the opposite way: I tend to prefer `it` for rather simple code (esp. one-liners) where there's no clash with another (nested) `it`. For the special case of `runCatching` I tend to only use a named lambda parameter for `onFailure` if also done so for an `onSuccess`.

I *do* see a small point here to use `e` to keep the diff a bit smaller, but then again `e` is not much more telling than `it` anyway (`exception` would have been more telling). So I chose to stick to `it`.

But I'm open to get overruled here. Any opinion from @oheger-bosch?

Reply:
I'm ok with `it`.

Reply:
IMHO we should not be too dogmatic, but give the developers a certain freedom to follow their preferred style. I am fine with both.